Source/Services/IConvertReverseCallMessages.cs

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Dolittle.Services.Contracts;
+
+namespace Dolittle.SDK.Services
+{
+    /// <summary>
+    /// Defines a converter that reads and writes parts of duplex streaming messages using the reverse call protocol.
+    /// </summary>
+    /// <typeparam name="TClientMessage">Type of the messages that is sent from the client to the server.</typeparam>
+    /// <typeparam name="TServerMessage">Type of the messages that is sent from the server to the client.</typeparam>
+    /// <typeparam name="TConnectArguments">Type of the arguments that are sent along with the initial Connect call.</typeparam>
+    /// <typeparam name="TConnectResponse">Type of the response that is received after the initial Connect call.</typeparam>
+    /// <typeparam name="TRequest">Type of the requests sent from the server to the client.</typeparam>
+    /// <typeparam name="TResponse">Type of the responses received from the client.</typeparam>
+    public interface IConvertReverseCallMessages<TClientMessage, TServerMessage, TConnectArguments, TConnectResponse, TRequest, TResponse>
```

Comment:
I think that we can think about this name. Doesn't really make sense considering what this represents